### PR TITLE
Libpitt/68 generalize

### PR DESCRIPTION
**Pull request by libpitt**
Change log:
https://github.com/hubmapconsortium/data-ingest-board/issues/62
https://github.com/hubmapconsortium/data-ingest-board/issues/63
https://github.com/hubmapconsortium/data-ingest-board/issues/64
https://github.com/hubmapconsortium/data-ingest-board/issues/67
https://github.com/hubmapconsortium/data-ingest-board/issues/68

There's a lot of changes. Can you @DerekFurstPitt @yuanzhou @maxsibilla help me test for bugs. 

I will set up HuBMAP's ingest-api in the meantime.

### CODE REVIEW

**Review by DerekFurstPitt — APPROVED** [review]

File: src/example.env [hunk 1]
```diff
@@ -1,6 +1,34 @@
 # Backend API services, shown for DEV
-NEXT_PUBLIC_APP_BACKEND_URL = 'https://ingest-api.dev.hubmapconsortium.org'
-NEXT_PUBLIC_DATASET_URL = 'https://ingest-api.dev.hubmapconsortium.org/datasets/data-status'
-NEXT_PUBLIC_UPLOAD_URL = 'https://ingest-api.dev.hubmapconsortium.org/uploads/data-status'
-NEXT_PUBLIC_PORTAL_URL = 'https://portal.dev.hubmapconsortium.org'
-NEXT_PUBLIC_INGEST_URL = 'https://ingest.dev.hubmapconsortium.org'
\ No newline at end of file
+NEXT_PUBLIC_API_BASE = https://ingest-api.dev.hubmapconsortium.org
+NEXT_PUBLIC_DATASET_URL = https://ingest-api.dev.hubmapconsortium.org/datasets/data-status
+NEXT_PUBLIC_UPLOAD_URL = https://ingest-api.dev.hubmapconsortium.org/uploads/data-status
```
Comment (yuanzhou) [inline src/example.env:4]: I had noticed this earlier when @DerekFurstPitt created the data-ingest-board. @libpitt It's a good opportunity now that we expand the config variable for reuse purpose as below:

```
NEXT_PUBLIC_API_BASE = https://ingest-api.dev.hubmapconsortium.org
NEXT_PUBLIC_DATASET_URL = $NEXT_PUBLIC_API_BASE/datasets/data-status
NEXT_PUBLIC_UPLOAD_URL = $NEXT_PUBLIC_API_BASE/uploads/data-status
```

File: src/package.json [hunk 2]
```diff
@@ -3,21 +3,34 @@
   "version": "1.1.0",
```
Comment (yuanzhou) [inline src/package.json:3]: @libpitt can you update the version to 1.2.0 in both `package.json` and the VERSION file?

**Review by yuanzhou — CHANGES_REQUESTED** [review]
@libpitt it looks great and I did some local testing too. In addition to the commented tweaks, can you also make one more change?

We really don't need the part after the seconds in the last modified column. I had pointed this out to @DerekFurstPitt and he mentioned something about sorting. You two may talk about the details. It takes more horizontal space and that decimal second doesn't really convery much meaning to the end users.

image

**Review by yuanzhou — APPROVED** [review]
@libpitt thanks for the tweaks! @maxsibilla once you also finish the review and approve, I'll get it merged and deployed on DEV along with the ingest-api PR: https://github.com/hubmapconsortium/ingest-api/pull/361

**Comment by maxsibilla** [comment]
I updated the README and added example.sennet.env file. @libpitt I think you should add some additional columns to be searchable such as:
- status
- organ type
- provider experiment ID

Also I am not sure the best way to handle this but the tooltip in this scenario should display the header name:
image

**Pull request merged**